### DISCUSSION TITLE
Use Auto-Adjusting Budgets

### DISCUSSION THREAD

**Discussion by sbstjn** [superwerker/superwerker#285] [Ideas]
AWS recently introduced [Auto-Adjusting Budgets](https://aws.amazon.com/about-aws/whats-new/2022/02/auto-adjusting-budgets/). This might be a nice replacement for the custom built budget adjustments.

**Comment by s0enke** [comment]
Thanks! The feature should be able to replace the custom-built Lambda-based logic. It's currently still missing CloudFormation support, which is hopefully added soon.

**Reply by mavogel** [reply]
Is available now in CFN: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-budgets-budget-autoadjustdata.html

**Comment by aperov9** [comment]
Done #351
